doc/modules/impute.rst

```diff
@@ -128,17 +130,36 @@ Multiple vs. Single Imputation
 ==============================
 
 In the statistics community, it is common practice to perform multiple imputations,
-generating, for example, 10 separate imputations for a single feature matrix.
-Each of these 10 imputations is then put through the subsequent analysis pipeline
-(e.g. feature engineering, clustering, regression, classification). The 10 final
+generating, for example, `m` separate imputations for a single feature matrix.
+Each of these `m` imputations is then put through the subsequent analysis pipeline
+(e.g. feature engineering, clustering, regression, classification). The `m` final
 analysis results (e.g. held-out validation error) allow the data scientist to
 obtain understanding of the uncertainty inherent in the missing values. The above
-practice is called multiple imputation. As implemented, the :class:`ChainedImputer`
-class generates a single (averaged) imputation for each missing value because this
-is the most common use case for machine learning applications. However, it can also be used
-for multiple imputations by applying it repeatedly to the same dataset with different
-random seeds with the ``n_imputations`` parameter set to 1.
+practice is called multiple imputation.
 
-Note that a call to the ``transform`` method of :class:`ChainedImputer` is not
+As implemented, the :class:`IterativeImputer` class generates a single imputation
```

Comment:
I would keep this more general on imputers in sklearn, as also the SimpleImputer (which would be relevant in the case of a future 'sampling' strategy) is a single imputation. 
Then in the next sentence you can replace 'it' with 'the IterativeImputer'

Reply:
Will do, thanks.